test/OpenTelemetry.Instrumentation.AWS.Tests/OpenTelemetry.Instrumentation.AWS.Tests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Unit test project for AWS client instrumentation for OpenTelemetry.</Description>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Seems the only remaining test project with netcoreapp3.1 is https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/test/OpenTelemetry.Instrumentation.GrpcCore.Tests/OpenTelemetry.Instrumentation.GrpcCore.Tests.csproj#L4. Should we also update the .NET version there ?

Reply:
It will be part of separate PR.
Probably whole package will be deprecated https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1031#issuecomment-1598368676

Reply:
Do you want to add `net462` as well?

Reply:
It is executed also on .net462.

```xml
    <TargetFrameworks>net6.0</TargetFrameworks>
    <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
```

I have did the same for main project 56a0e0f1fd26f92586ac825ecbcf68bda6b2ad3b